generation/WinSDK/emitter.settings.rsp

```diff
@@ -49,7 +49,6 @@ DELETE_SNAPSHOT_VHDSET_FLAG
 DEPENDENT_DISK_FLAG
 DETACH_VIRTUAL_DISK_FLAG
 DRAWPROGRESSFLAGS
-DXGI_ADAPTER_FLAG
```

Comment:
I moved this into `generation/WinSDK/Partitions/Dxgi/settings.rsp` with `--enumMakeFlags` but the `[Flags]` attribute disappears. Is it only expected to work inside `emitter.settings.rsp`? Other flags (`--with-type`, `--with-attribute` do work there).

What's preferred:
1. Have the types under `--enumMakeFlags` in `emitter.settings.rsp`
2. Have `TYPE=Flags` under `--with-atribute` in the `Dxgi`-specific `settings.rsp` above?

Reply:
`--memberRemap` also doesn't seem to have an effect in `generation/WinSDK/Partitions/Dxgi/settings.rsp`, only in `generation/WinSDK/emitter.settings.rsp`.